src/Veldrid/OpenGL/OpenGLCommandExecutor.cs

```diff
@@ -964,14 +964,16 @@ public void SetComputeResourceSet(uint slot, ResourceSet rs, uint dynamicOffsetC
                             else
                             {
                                 glBindImageTexture(
-                                    (uint)imageBindingInfo.UniformLocation,
+                                    (uint)imageBindingInfo.RelativeIndex,
                                     glTexViewRW.Target.Texture,
                                     (int)texViewRW.BaseMipLevel,
                                     layered,
                                     (int)texViewRW.BaseArrayLayer,
                                     TextureAccess.ReadWrite,
                                     glTexViewRW.GetReadWriteSizedInternalFormat());
                                 CheckLastError();
+                                glUniform1i(imageBindingInfo.UniformLocation, imageBindingInfo.RelativeIndex);
+                                CheckLastError();
```

Comment:
[This commit](https://github.com/mellinoe/veldrid/commit/77588d4b6c1660be09ef2d4553610738f993a498#diff-c1c403b65fcd771b421112856cdf932bd732728f4a16267f99523ffbf6614bd8R631) appears to have split this implementation, but without this update my sample breaks.  Is there a way I can repro the problem that caused the divergence?